spec/models/events/creator_spec.rb

```diff
@@ -1,6 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Events::Creator, :type => :model do
+
+  before do
+    Timecop.freeze(Time.local(1990))
```

Comment:
Yes. Line 36 would have small diffs (due to database latency) and flake occasionally. 

Additionally, you typically don't test stuff like that as it's a feature of the framework  (specifically ActiveRecord) anyways.